sdk/core/core/src/az_http_response_parser.c

```diff
@@ -298,3 +298,32 @@ az_http_response_get_body(az_span const self, az_pair * const p_last_header, az_
   AZ_RETURN_IF_FAILED(az_http_response_parser_get_body(&parser, body));
   return AZ_OK;
 }
+
+/**
+ * Get an HTTP header by name.
+ *
+ * Returns @AZ_ERROR_HTTP_NO_MORE_ITEMS
+ */
+AZ_NODISCARD az_result az_http_response_get_header_by_name(
+    az_span const self,
+    az_span const header_name,
+    az_span * const header_value) {
+  AZ_CONTRACT_ARG_NOT_NULL(header_value);
+
+  az_http_response_parser parser;
+  AZ_RETURN_IF_FAILED(az_http_response_parser_init(&parser, self));
+
```

Comment:
NIT, Extra whitespace

Reply:
I would like to keep them together like a constuctor. Initialize ASAP.